hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/UmbrellaRedux.java

```diff
@@ -55,18 +50,15 @@ public class UmbrellaRedux {
     @HapiTest
     @Tag(NOT_REPEATABLE)
     final Stream<DynamicTest> umbrellaRedux() {
```

Comment:
I have no idea what an `umbrellaRedux` test is about so I have no idea why it _used to_ update the throttles with the default (now we see that `-dev` **does mean** "default" after all) and now it doesn't have to.  Maybe some javadoc somewhere would help explain what the heck this test is for?

Reply:
There are a few of these `RegressionProviderFactory`-based tests...they have some uses, but are quite clumsy (and yes, undocumented!)...created an [issue](https://github.com/hashgraph/hedera-services/issues/14850) to evaluate the ROI of this testing approach.